src/systems/filecoin_markets/storage_market/storage_market_actor_code.go

```diff
@@ -72,7 +85,7 @@ func (a *StorageMarketActorCode_I) WithdrawBalance(rt Runtime, balance actor.Tok
 	senderBalance.Impl().Available_ = senderBalance.Available() - balance
 	st.Balances()[msgSender] = senderBalance
 
-	// TODO send funds to msgSender with `transferBalance` in VM runtime
```

Comment:
we are using a different transferBalance here in rt.Send(). This is left as a TODO for future PRs.